INSTALLING.md

```diff
@@ -4,197 +4,271 @@ These instructions will guide you through the installation of Tenstorrent system
 
-```sh
-sudo -E python3 setup_hugepages.py enable && sudo -E python3 setup_hugepages.py check
+---
+
+#### Install System Management Interface (TT-SMI)
+- Install Tenstorrent Software Management Interface (TT-SMI):
+```
+pip install git+https://github.com/tenstorrent/tt-smi
 ```
+
+- Verify System Configuration
+
+Once hardware and system software are installed, verify that the system has been configured correctly.
+
+  - Run the TT-SMI utility:
+  ```
+  tt-smi
+  ```
+  A display with device information, telemetry, and firmware will appear:<br>
+
+![image](https://docs.tenstorrent.com/_images/tt_smi.png)
+<br>
+  If the tool runs without error, your system has been configured correctly.
+
+- For more information, visit Tenstorrent's [TT-SMI GitHub repository](https://github.com/tenstorrent/tt-smi).
+
 ---
 
-### Step 4. Install and start using TT-NN and TT-Metalium!
+#### (Optional) Multi-Card Configuration (TT-Topology)
+- For TT-Loudbox or TT-QuietBox systems, visit Tenstorrent's [TT-Topology README](https://github.com/tenstorrent/tt-topology/blob/main/README.md).
 
-> [!NOTE]
->
-> You may choose to install from either source, a Python wheel, or Docker release image.
->
-> However, no matter your method, in order to use our pre-built models or to
-> follow along with the documentation and tutorials to get started, you will
-> still need the source code.
->
-> If you do not want to use the models or follow the tutorials and want to
-> immediately start using the API, you may install just the wheel or get the release Docker container.
+---
 
-1. Install git and git-lfs.
+## TT-Metalium Installation
 
-```sh
-sudo apt install git git-lfs
-```
+#### There are three options for installing TT-Metalium:
+
+- [Option 1: From Source](#option-1-from-source)
+  Installing from source gets developers closer to the metal and the source code.
+
+- [Option 2: From Docker Release Image](#option-2-from-docker-release-image)
+  Installing from Docker Release Image is the quickest way to access our APIs and to start runnig AI models.
+
+- [Option 3: From Wheel](#option-3-from-wheel)
+  Install from wheel as an alternative method to get quick access to our APIs and to running AI models.
+
+--- 
 
-2. Clone the repo.
+## Option 1: From Source
+Install from source if you are a developer who wants to be close to the metal or the source code.
+
+### Step 1. Clone the Repository:
 
 ```sh
 git clone https://github.com/tenstorrent/tt-metal.git --recurse-submodules
 cd tt-metal
 git submodule foreach 'git lfs fetch --all && git lfs pull'
 ```
 
-3. Install either from source, or from our release wheel. Note that if you are
-going to try using the model demos, we highly recommend you install from
-source.
+### Step 2. Setup Environment Variables and Invoke our Build Scripts:
 
-#### Option 1: From source
+- Note: Some advanced build configurations like unity builds require `CMake 3.20`.
+  To upgrade Cmake run:
+    ```sh
+    pip3 install cmake --upgrade
```

Comment:
Why are we installing `cmake` into the python environment?

I would slightly prefer if it's a system dependency.

Reply:
This method seemed to work easier (in VMs at least). Is this the script we should refer to? 

```
# Use a newer version of CMake than what is available from Canonical for 20.04
RUN apt -y update \
    && apt install -y --no-install-recommends ca-certificates gpg wget \
    && wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null \
    && echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ focal main' | tee /etc/apt/sources.list.d/kitware.list >/dev/null \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
"Work easier"?

This might conflict with `install_dependencies.sh`, and also note that not all users will be in a Python environment, like certain customers we have in mind.

Please use `apt`

Reply:
What I mean by "work easier" is that we were able to successfully install it with fewer commands and running into less issues during tests. I have no technical preference in this matter and will include the method your prefer. Just please let me know if the commands I included above are satisfactory.

Reply:
I prefer we do not use `pip` install.

Try a

```
sudo apt update
sudo apt install cmake
```

to get newest cmake

Reply:
FWIW we install cmake with apt install so it would follow what our current processes are as well

Reply:
Thanks guys, update it in latest commit.